Userland/Applications/Browser/main.cpp

```diff
@@ -57,6 +59,26 @@ static ErrorOr<void> load_content_filters()
     return {};
 }
 
+static ErrorOr<void> load_autoplay_allowlist()
+{
+    auto file = TRY(Core::File::open(DeprecatedString::formatted("{}/BrowserAutoplayAllowlist.txt", Core::StandardPaths::config_directory()), Core::File::OpenMode::Read));
```

Comment:
```suggestion
    auto file = TRY(Core::File::open(TRY(String::formatted("{}/BrowserAutoplayAllowlist.txt", Core::StandardPaths::config_directory())), Core::File::OpenMode::Read));
```
You use `String::Formatted` for `Core::File::open` elsewhere, and it's already inside a failable function.

Reply:
Thanks! Missed this one when copying this code from elsewhere, fixed.